src/main/java/com/squareup/javapoet/AnnotationSpec.java

```diff
@@ -165,6 +166,15 @@ public static AnnotationSpec get(AnnotationMirror annotation) {
     return builder.build();
   }
 
+  public static AnnotationSpec createSimpleAnnotation(ClassName type, Object value) {
```

Comment:
May be enhanced to take `Object... values` for producing:
 * tagging annotation (without value) like simply `@AnnotationD`: `createSimpleAnnotation(AnnotationD.class)`
 * and array of values like `@AnnotationD({"a", "b", "c"})`: `createSimpleAnnotation(AnnotationD.class, "a", "b", "c")`

with 
```java
  @Retention(RetentionPolicy.RUNTIME)
  public @interface AnnotationD {
    String[] value() default {"x", "y", "z"};
  }
```